bucket/bruno.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "1.2.0",
+    "homepage": "https://www.usebruno.com/",
+    "license": "MIT",
+    "description": "Open source IDE for exploring and testing APIs (lightweight alternative to Postman/Insomnia)",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/usebruno/bruno/releases/download/v1.2.0/bruno_1.2.0_x64_win.zip",
+            "hash": "98941b455b71d7c8c8bfff5c448c0587960962d947a2e37665e411bffb95d4d0",
+            "extract_dir": "win-unpacked"
+        }
+    },
+    "shortcuts": [
+        [
+            "Bruno.exe",
+            "Bruno"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/usebruno/bruno"
```

```suggestion
            "hash": "98941b455b71d7c8c8bfff5c448c0587960962d947a2e37665e411bffb95d4d0"
        }
    },
    "extract_dir": "win-unpacked",
    "shortcuts": [
        [
            "Bruno.exe",
            "Bruno"
        ]
    ],
    "checkver": {
        "github": "https://github.com/usebruno/bruno"
```


Almost there, could you move `extract_dir` out of architecture and put it before `shortcuts` as per https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets ? That way if there's a new architecture in the future, the same directory can be used to extract for both.